docs/docs/cloud/cloud-cli.mdx

```diff
@@ -48,6 +48,101 @@ meltano cloud config env delete TAP_GITHUB_AUTH_TOKEN
 
 See the [reserved variables](/cloud/platform#reserved-variables) docs for more details on variables that are reserved for use by Meltano Cloud.
 
+### `notification`
+
+The `notification` subcommand provides a way for managing your [Meltano Cloud notifications](concepts#meltano-cloud-notifications).
+
+#### `list`
+
+List all notifications configured for your current default project or organization.
+
+```sh
+$ meltano cloud config notification list
+╭──────────────────────────┬─────────┬──────────┬─────────────────────────────────────────╮
+│ Recipient                │ Type    │ Status   │ Filters                                 │
+├──────────────────────────┼─────────┼──────────┼─────────────────────────────────────────┤
+│ dragon@meltano.com       │ email   │ active   │ N/A                                     │
+│ https://melty            │ webhook │ active   │ [{'events': ['job_run'], 'status': []}] │
+╰──────────────────────────┴─────────┴──────────┴─────────────────────────────────────────╯
+```
+
+#### **Notification System Limitations**
+
+The notification system has a few limitations that should be taken into consideration:
+
+- **Unique Notifications**: Currently, our system permits only a single notification per recipient within each scope, 'organization' or 'project'.
+- **Notification Overwrite**: If you attempt to set another notification for the same recipient within the same scope, the new notification will overwrite the existing one. In other words, setting a second notification won't create a new notification, but rather modify the existing one.
+
+#### `set`
+
+Set a new notification or override an existing one.
+
+##### Case-Sensitivity in Values
+
+Take note that all the values in the commands and configuration settings are case-sensitive. This means job_created, Job_Created, and JOB_CREATED would be considered as different events. Please ensure to use the correct casing to avoid configuration issues or unexpected behaviors.
+
+
+Interactive mode
+```sh
+meltano cloud config notification set <type>
+```
+
+Non-interactive mode:
+```sh
+meltano cloud config notification set <type> --recipient <target_recipient> --filter <filter>
+````
+
+**`options`**
+
+`--recipient` (Required): The recipient to send the notification to.
+
+`--filter`: An optional filter you want to check against, you can supply multiple filters which will all be joined as an OR expression.
+Values within the filter are validated as an AND Expression.
+
+**Job Failures**
+
+To subscribe only to job failures you would run
+```sh
+meltano cloud config notification set webhook --recipient http://test+melty --filter '{"events": ["job_run"], "status": ["failed"]}'
```

Comment:
@leoroese I'm running this exact example and am getting the following:

```bash
❯ meltano-cloud config notification set email --recipient taylor2@meltano.com --filter '{"events": ["job_run"], "status": ["failed"]}'
2023-08-22 16:51:54 [debug    ] Making Cloud CLI HTTP request  method=POST url=https://internal.api.meltano.cloud/notifications/v1/01GV0Z1VW5TSRF8BYX1J2V3WNT/cf6fc79ab63d9f69309059ecea8866416df7372ec1accbe1e103518730f24ca7/notification?entity_id=01H8D07597KR2Y474SSJG3CTT2
Error: . 

Please ensure you've correctly followed instructions at: https://docs.meltano.com/cloud/cloud-cli
If the issue persists, we'd be happy to help at: https://meltano.com/slack
```

Reply:
Taking a look

Reply:
<img width="1491" alt="Screenshot 2023-08-22 at 5 19 21 PM" src="https://github.com/meltano/meltano/assets/14004680/f53e1364-e0d7-48d2-97ea-f820770ccc72">